ksql-engine/src/test/resources/SampleMultilineStatements.sql

```diff
@@ -0,0 +1,13 @@
+CREATE STREAM pageviews_original (viewtime bigint, userid varchar, pageid varchar) WITH (kafka_topic='pageviews', value_format='DELIMITED');
```

Comment:
This should contain all the commands that are supported. I'm not sure if any others are missing, but DROP looks to be missing.

Reply:
Now that we are deprecating `RUN SCRIPT` this won't need to run `DROP` and `TERMINATE` statements. It just needs to run the statements that we have in `functionMap` in `StandaloneExecutor` class.

Reply:
hmm! I see you mean, that would be a great test to have. Don't know how we can test that we have a test for each member of functions map. Any suggestions?